src/iosrv/srv.c

```diff
@@ -792,27 +795,28 @@ dss_start_one_xstream(hwloc_cpuset_t cpus, int xs_id)
 
 	rc = dss_sched_create(dx->dx_pools, DSS_POOL_CNT, &dx->dx_sched);
 	if (rc != 0) {
-		D_ERROR("create scheduler fails: %d\n", rc);
+		D_ERROR("create scheduler fails: "DF_RC"\n", DP_RC(rc));
 		D_GOTO(out_pool, rc);
 	}
 
 	/** start XS, ABT rank 0 is reserved for the primary xstream */
 	rc = ABT_xstream_create_with_rank(dx->dx_sched, xs_id + 1,
 					  &dx->dx_xstream);
 	if (rc != ABT_SUCCESS) {
-		D_ERROR("create xstream fails %d\n", rc);
+		D_ERROR("create xstream fails "DF_RC"\n", DP_RC(rc));
 		D_GOTO(out_sched, rc = dss_abterr2der(rc));
 	}
 
 	rc = ABT_thread_attr_create(&attr);
 	if (rc != ABT_SUCCESS) {
-		D_ERROR("ABT_thread_attr_create fails %d\n", rc);
+		D_ERROR("ABT_thread_attr_create fails "DF_RC"\n", DP_RC(rc));
```

Comment:
Logs of ABT_ calls that are not daos errors.   Should either print rc directly or use dss_abterr2derr first

Reply:
done